src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -132,3 +198,47 @@
             :on-change      on-tab-change
             :data           tabs-data}]
           [tabs/view {:selected-tab @selected-tab}]]]))))
+
+(defn view
+  [_]
+  (let [title-ref          (atom nil)
+        set-title-ref      #(reset! title-ref %)
+        title-bottom-coord (atom 0)
```

Comment:
If we need atoms, Can we please use the `rn/use-ref-atom` helper as declared in the guidelines?
## Regular atoms

In certain instances, components utilized regular atoms; however, they should now be used with `rn/use-ref-atom`

BEFORE:
```clojure
(defn view
  []
  (let [focused? (atom false)]
    (fn []
      (let [on-clear #(reset! status (if @focused? :active :default))
            on-focus #(reset! focused? true)
            on-blur  #(reset! focused? false)]))))
```

NOW:
```clojure
(defn view
  []
  (let [focused? (rn/use-ref-atom false)
        on-clear (rn/use-callback #(set-status (if @focused? :active :default)))
        on-focus (rn/use-callback #(reset! focused? true))
        on-blur  (rn/use-callback #(reset! focused? false))]))

Reply:
@ibrkhalil Thanks for pointing this out.

I moved the atoms I created to `use-ref-atom`. TBH now I feel the code more verbose and harder to understand, but it's on the guidelines.